compiler/luci/pass/src/SubstituteSqueezeToReshapePass.cpp

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/Pass/SubstituteSqueezeToReshapePass.h"
+
+#include <luci/IR/CircleNodes.h>
+
+namespace
+{
+
+/**
+ * @brief return TRUE if all dim is known
```

Comment:
There are some cases that this pass can be applied even some of dimensions are unknown.
```python
p = tf.compat.v1.placeholder(shape=[None, 1, 2], dtype=tf.int32)
s = tf.squeeze(p,[0])
print(s) # <tf.Tensor 'Squeeze:0' shape=(1, 2) dtype=int32>
```

However, same logic may be also used for shape inference of `CircleSqueeze`.

So how about going as it is and just adding following comments?
```suggestion
 * @brief return TRUE if all dim is known
 * @note This pass can be applied even some of dimensions are unknown.
         For now, do not consider about it and update logic later.
```